<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>718</width>
    <height>492</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>1</horstretch>
    <verstretch>1</verstretch>
   </sizepolicy>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QRadioButton{
padding-left: 10px;
text-align:center;
border:1px solid black;
}
QRadioButton:hover{
border:1px solid red;
}
QRadioButton:checked{
border:5px solid red;
}

QRadioButton::indicator {
    image: none;
}
QMessageBox {
	height: 200px !important;
	width: 300px !important;
	border-radius: 10px;
   background-color: #3d3d3d;
}

QMessageBox QLabel {
    color: #fff !important;
}

QToolTip{
color: #aaa !important;
}

</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color:lightgrey;
padding: 0;</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QFrame" name="frame">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>1600000</width>
        <height>50</height>
       </size>
      </property>
      <property name="cursor">
       <cursorShape>ArrowCursor</cursorShape>
      </property>
      <property name="mouseTracking">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 10px;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QRadioButton" name="dotButton">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="mouseTracking">
          <bool>true</bool>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="icons/line.qrc">
           <normaloff>:/icons/dot.png</normaloff>:/icons/dot.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <property name="autoExclusive">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="lineButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="icons/line.qrc">
           <normaloff>:/icons/line.png</normaloff>:/icons/line.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="ellipseButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="icons/line.qrc">
           <normaloff>:/icons/ellipse.png</normaloff>:/icons/ellipse.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="rectButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="palette">
          <palette>
           <active>
            <colorrole role="Button">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>211</red>
               <green>211</green>
               <blue>211</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Base">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>211</red>
               <green>211</green>
               <blue>211</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Window">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>211</red>
               <green>211</green>
               <blue>211</blue>
              </color>
             </brush>
            </colorrole>
           </active>
           <inactive>
            <colorrole role="Button">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>211</red>
               <green>211</green>
               <blue>211</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Base">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>211</red>
               <green>211</green>
               <blue>211</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Window">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>211</red>
               <green>211</green>
               <blue>211</blue>
              </color>
             </brush>
            </colorrole>
           </inactive>
           <disabled>
            <colorrole role="Button">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>211</red>
               <green>211</green>
               <blue>211</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Base">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>211</red>
               <green>211</green>
               <blue>211</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Window">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>211</red>
               <green>211</green>
               <blue>211</blue>
              </color>
             </brush>
            </colorrole>
           </disabled>
          </palette>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="icons/line.qrc">
           <normaloff>:/icons/702814-200.png</normaloff>:/icons/702814-200.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoRepeat">
          <bool>false</bool>
         </property>
         <property name="autoExclusive">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="dumbbellButton">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="icons/line.qrc">
           <normaloff>:/icons/dumbbell_206315.png</normaloff>:/icons/dumbbell_206315.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="cubeButton">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="icons/line.qrc">
           <normaloff>:/icons/645891.png</normaloff>:/icons/645891.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>40</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>718</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_2">
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Об'єкти</string>
    </property>
    <addaction name="dot_edit"/>
    <addaction name="line_edit"/>
    <addaction name="ellipse_edit"/>
    <addaction name="rect_edit"/>
    <addaction name="dumbbell_edit"/>
    <addaction name="cube_edit"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Дії</string>
    </property>
    <addaction name="action_Toolbar"/>
    <addaction name="action"/>
    <addaction name="action_2"/>
   </widget>
   <addaction name="menu_2"/>
   <addaction name="menu"/>
  </widget>
  <action name="action_Toolbar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Сховати toolbar</string>
   </property>
  </action>
  <action name="dumbbell_edit">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Гантеля</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="cube_edit">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Кубаноїд</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>Показати таблицю</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>Зберегти</string>
   </property>
  </action>
  <actiongroup name="GroupTool">
   <action name="line_edit">
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>Лінія</string>
    </property>
   </action>
   <action name="rect_edit">
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>Прямокутник</string>
    </property>
   </action>
   <action name="ellipse_edit">
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>Еліпс</string>
    </property>
   </action>
   <action name="dot_edit">
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>Крапка</string>
    </property>
   </action>
  </actiongroup>
 </widget>
 <resources>
  <include location="icons/line.qrc"/>
 </resources>
 <connections/>
</ui>
